metastore/iceberg-metastore/src/main/java/org/apache/drill/metastore/iceberg/IcebergMetastore.java

```diff
@@ -75,14 +77,15 @@ public IcebergMetastore(DrillConfig config) {
     this.tables = new HadoopTables(new Configuration(configuration));
```

Comment:
looks like there is no need to pass ```config``` into ```configuration(config)```  , ```baseLocation(config, ...);``` and ```properties(config, ...)``` because it's usage of field set at first line in constructor.